src/mocks/server_request.ts

```diff
@@ -29,7 +43,7 @@ export const MockServerRequest = function (
   //
   //   TypeError: Cannot read property 'write' of undefined
   //
-  request.respond = function respond(output: any) {
+  request.respond = function respond(output: any) { // TODO RequestRespondOutput & different method signature than original
```

Comment:
Did not found a usecase test, dont know what should be done with this.

Reply:
Oh, i think theres a unit test in `mod_test.ts` (or somewhere else) that uses that object - i believe the it contains `{ status: 200 }`, but need to check with @crookse if it should contains properties a HTTP request responds with

Reply:
Well, my problem is, that the default ServerRespond is async, this changes it to a sync method. If you can give me some usecases, i think I can type it, but I dont want to have a totally different type for it, than it should

Reply:
we can trash this. this code was taking from drash

Reply:
Removed `request.respond` overwriting